wallet-extension/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "wallet-extension",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "build:general": "vite build",
+    "build": "tsc && npm run build:general",
+    "lint": "biome check .",
+    "lint:all": "npm run lint && npm run lint:types && npm run lint:jsdoc",
+    "lint:fix": "biome check --write .",
+    "lint:types": "tsc --noEmit -p tsconfig.json",
+    "lint:jsdoc": "eslint .",
+    "interface": "i18next-resources-for-ts interface -i public/locales/en -o src/@types/resources.d.ts"
+  },
+  "dependencies": {
+    "@nilfoundation/niljs": "^0.22.0",
```

Comment:
this will also be reused form the local repo which is quite useful for development purposes. and remove `^`  cause currently we support only latest versions of the packages.